--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,6 +6,7 @@

 [dependencies]
 alsa = { version = "0.3", optional = true }
+cc = "1.0.47"
 chrono = "0.4"
 daemonize = "0.4"
 dbus = { version = "0.6", optional = true }
@@ -32,7 +33,16 @@
 url = "1.7"
 xdg = "2.2"
 whoami = "0.7.0"
-librespot = { version = "0.1", default-features = false, features = ["with-tremor"] }
+
+[dependencies.librespot]
+path = "../librespot-0.1.0"
+default-features = false
+features = ["with-tremor"]
+version = "0.1.1"
+
+[dependencies.structopt]
+features = ["color"]
+version = "0.3.2"

 [dev-dependencies]
 env_logger = "0.7"
