--- Slim/Player/Playlist.pm.orig	2012-05-29 13:43:52.000000000 +0100
+++ Slim/Player/Playlist.pm	2012-05-29 13:43:52.000000000 +0100
@@ -1189,7 +1189,7 @@ sub _playlistUrlForClient {
        $id =~ s/://g;

        return Slim::Utils::Misc::fileURLFromPath(
-               catfile(scalar Slim::Utils::OSDetect::dirsFor('prefs'), "client>
+               catfile(Slim::Utils::OSDetect::dirsFor('ClientPlaylists'), "cli>
        );
 }
