diff --git a/slimserver.pl b/slimserver.pl
index 5a81cb8..43596f8 100755
--- a/slimserver.pl
+++ b/slimserver.pl
@@ -477,8 +477,9 @@ sub init {

        # Generate a UUID for this SC instance on first-run
        if ( !$prefs->get('server_uuid') ) {
-               require UUID::Tiny;
-               $prefs->set( server_uuid => UUID::Tiny::create_UUID_as_string( >
+               require Data::UUID;
+               my $ug = new Data::UUID;
+               $prefs->set( server_uuid => lc $ug->create_str() );
        }

        main::INFOLOG && $log->info("Server binary search path init...");
