diff --git a/Playlist.pm b/Playlist1.pm
index f45a577..e3ff25c 100644
--- a/Slim/Player/Playlist.pm.orig
+++ b/Slim/Player/Playlist.pm
@@ -1189,7 +1189,7 @@ sub _playlistUrlForClient {
        $id =~ s/://g;

        return Slim::Utils::Misc::fileURLFromPath(
-               catfile(scalar Slim::Utils::OSDetect::dirsFor('prefs'), "client>
+               catfile(Slim::Utils::OSDetect::dirsFor('ClientPlaylists'), "cli>
        );
 }
